src/test/jtx/impl/mpt.cpp

```diff
@@ -163,22 +169,37 @@ MPTTester::authorize(MPTAuthorize const& arg)
         jv[sfMPTokenHolder.jsonName] = arg.holder->human();
     if (submit(arg, jv) == tesSUCCESS)
     {
+        // Issuer authorizes
         if (arg.account == nullptr || *arg.account == issuer_)
         {
+            auto const flags = getFlags(arg.holder);
             // issuer un-authorizes the holder
             if (arg.flags.value_or(0) == tfMPTUnauthorize)
-                env_.require(mptflags(*this, 0, arg.holder));
+                env_.require(mptflags(*this, flags, arg.holder));
```

Comment:
Is the purpose of this change to account for the possibility that the MPToken object might also have lsfMPTLocked set?

Reply:
Correct.